src/navigation/rootNavigation.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createDrawerNavigator } from "@react-navigation/drawer";
+
+import PlaceholderComponent from "../components/PlaceholderComponent";
+import ObservationProvider from "../providers/ObservationProvider";
+import StackNavigator from "./stackNavigation";
+
+// this removes the default hamburger menu from header
+const screenOptions = { headerLeft: ( ) => <></> };
+const hideHeader = {
+  headerShown: false,
+  label: "my observations"
+};
+
+const Drawer = createDrawerNavigator( );
+
+const App = ( ): React.Node => (
+  <NavigationContainer>
+    <ObservationProvider>
```

Comment:
If we provide the user's observations to the entire app, does that mean they're hanging around in state even if they're spending all their time in Explore or Identify? E.g. you go to My Observations and your obs get loaded from Realm and added to the global state where they're available to My Observations as well as obs detail. But then you leave and go to Identify: are your local observations still hanging around in memory even though you're not using them any more?

Reply:
Yep, I'm pretty sure they would still be in memory. Maybe a better (more performant) way to do this is to create one context and provider for My Observations and a separate one for Explore.

Reply:
Yeah, I think that sounds right, especially considering they'll be serving different kinds of data (presumably): dynamic data from the API in Explore, local data from Realm in My Observations (I think).